proposals/0085-additional-fee-collector-constraints.md

```diff
@@ -0,0 +1,61 @@
+---
+simd: "0085"
+title: Additional Fee Collector Constraints
+authors:
+  - Justin Starry
+category: Standard
+type: Core
+status: Draft
+created: 2023-11-05
+feature: https://github.com/solana-labs/solana/issues/33888
+---
+
+## Summary
+
+Burns fees if they would otherwise be collected into an account that violates
+one of the following constraints:
+
+1. Must be a system owned account
+2. Must be rent-exempt
```

Comment:
SIMD-0084 aims to disable rent. Do you want to just

Reply:
In order to disable rent collection, all accounts must be enforced to be rent exempt when created. Fee collection was discovered to be an exception because it modifies accounts when depositing collected fees and didn't enforce that the new balance is rent exempt. So this exception needs to be patched to disable rent.